### PR TITLE
Base Card, PDF417 and MRZ Scanners

### PR DESCRIPTION
**Pull request by Juliano1612**
This PR has the following:
- generic multi-credential base card (+ credential pack)
- a generic PDF417 scanner 
- an initial MRZ (OCR) scanner based on the first QR Code scanner implementation
- refactors the previous QR Code Scanner version

### CODE REVIEW

File: Sources/MobileSdk/CredentialPack.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        do {
+            let mdocData = Data(base64Encoded: mdocBase64)!
+            let key = try P256.Signing.PrivateKey(pemRepresentation: keyPEM)
+            let attributes = [kSecAttrKeyType: kSecAttrKeyTypeECSECPrimeRandom,
+                             kSecAttrKeyClass: kSecAttrKeyClassPrivate] as [String: Any]
+            let secKey = SecKeyCreateWithData(key.x963Representation as CFData,
+                                              attributes as CFDictionary,
+                                              nil)!
+            let query = [kSecClass: kSecClassKey,
+          kSecAttrApplicationLabel: "mdoc_key",
```
Comment (w4ll3) [inline Sources/MobileSdk/CredentialPack.swift]: should this be hardcoded?

File: Sources/MobileSdk/GenericJSON.swift [hunk 2]
```diff
@@ -0,0 +1,139 @@
+// GenericJSON implementation based on https://github.com/iwill/generic-json-swift
```
Comment (w4ll3) [inline Sources/MobileSdk/GenericJSON.swift:1]:
```suggestion
```

File: Sources/MobileSdk/CredentialPack.swift (hunk 1 above)
Comment (Juliano1612) [reply Sources/MobileSdk/CredentialPack.swift]: No. Good catch!

**Review by w4ll3 — APPROVED** [review]
lgtm

**Pull request merged**